atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -61,6 +61,30 @@ protected final <T, E extends Exception> T runTaskThrowOnConflict(TransactionTas
         }
     }
 
+    protected final <T, E extends Exception> T runTaskThrowOnConflictWithCallback(
```

Comment:
There is a method above this with the same implementation except without commitWithoutCallbacks. However, that's not used within AtlasDB, and have tried searching for uses internally (but not totally convinced that the search was sufficient). I found no uses (as this is different from runTaskThrowOnConflict(TransactionTask<T, E> task), as this has an extra parameter.

If we do want to keep both, we _could_ deduplicate by factoring out the commit method. (although that may make this less readable)